script/cadence/scripts/get-keys-length.cdc

```diff
@@ -1,4 +1,4 @@
-pub fun main(addr: Address): Int {
+access(all) fun main(addr: Address): Int {
     let acct = getAccount(addr)
```

Comment:
Not a blocker, but this could all be simplified to
```cadence
    return getAccount(addr).keys.count
```
ref: https://cadence-lang.org/docs/1.0/language/accounts/keys#accountkeys
e.g. https://run.dnz.dev/snippet/5fbf4a3d0015785e